lib/airbrake/better_errors.rb

```diff
@@ -0,0 +1,38 @@
+require 'better_errors'
+
+module BetterErrors
+  class Middleware
+    alias_method :show_error_page_original, :show_error_page
+
+    private
+
+    def show_error_page(env, exception=nil)
+      env['airbrake.error_id'] = notify_airbrake(env, exception)
+      show_error_page_original(env, exception)
+    end
+
+    def notify_airbrake(env, exception)
+      unless ignored_user_agent? env
+        Airbrake.notify_or_ignore(exception, request_data(env))
+      end
+    end
+
+    def ignored_user_agent?(env)
```

Comment:
Dunno. Probably due to some historical reasons that I am not aware of.
I guess it can be changed like that, but not in this PR.